.ansible-lint

```diff
@@ -31,7 +31,7 @@ mock_modules:
   - community.general.archive
   - community.general.yum_versionlock
   - community.general.rhsm_repository
-  - ansible.posix.selinux
+  - redhat.rhel_system_roles.selinux
```

Comment:
```suggestion
```
We cannot use the module.  We must use the role.

Reply:
For selinux yes, we should use the role to ensure that selinux is properly configured. I guess it makes sense to use fedora.linux_system_roles.selinux role in the upstream. We will change this in RPM spec for downstream.

How about `ansible.posix.mount`? Downstream, is it a good idea to use `redhat.rhel_system_roles.mount` to avoid re-vendoring it?